README.md

```diff
@@ -0,0 +1,45 @@
+# cypress-utils-criteo
+
+This packages offers custom Cypress commands, recommended checks, and Angular Material selectors to ease the development of Cypress tests.
+
+## Usage
+
+Install this package `npm install cypress-utils-criteo --save-dev`
+
+If you use Typescript: in the `tsconfig.json` file used by Cypress add the types definition: `"types": ["cypress", "cypress-utils-criteo"]` (under `compilerOptions`)
+
+Depending on what you want to use:
+
+For **commands**, in `cypress/support/commands.js`, add:
+
+- `import 'cypress-utils-criteo/commands';` to import all exposed commands
+- `import 'cypress-utils-criteo/commands/selectors';` to import specific ones
+
+For **recommended checks**, in `cypress/support/e2e.js`, add:
+
+- `import 'cypress-utils-criteo/recommended-checks';` to import all recommended checks
+- `import 'cypress-utils-criteo/recommended-checks/no-open-mat-snack-bar';` to import specific ones
+
+## Commands
+
+### `selectors/getByTestAttr`
```

Comment:
Do we need this documentation? Will we keep it up to date? 
If yes, maybe we could move it to the commands folder directly?

Reply:
Since this is an open source lib, I think it is really important to provide good and up-to-date documentation. The README file is the one highlighted by Github and as developers, we are used to read it before installing a new package. Maybe when this package will contain too many stuff, we will be able to split the doc then?

Reply:
OK, agreed